server/odc-service/src/main/java/com/oceanbase/odc/service/connection/ConnectionService.java

```diff
@@ -238,21 +242,30 @@ public ConnectionConfig create(@NotNull @Valid ConnectionConfig connection, @Not
             transactionManager.rollback(transactionStatus);
             throw e;
         }
-        databaseSyncManager.submitSyncDataSourceTask(saved);
+        databaseSyncManager.submitSyncDataSourceAndDBSchemaTask(saved);
         return saved;
     }
 
-    @Transactional(rollbackFor = Exception.class)
     @PreAuthenticate(actions = "create", resourceType = "ODC_CONNECTION", isForAll = true)
     public List<ConnectionConfig> batchCreate(@NotEmpty @Valid List<ConnectionConfig> connections) {
+        TransactionDefinition transactionDefinition = new DefaultTransactionDefinition();
```

Comment:
tips: TransactionTemplate  make tx control simpler

Reply:
Thx for your suggestion, I refined it.